docker/.env

```diff
@@ -15,16 +15,30 @@
 # limitations under the License.
 #
 COMPOSE_PROJECT_NAME=superset
-POSTGRES_DB=superset
-POSTGRES_HOST=postgres
-POSTGRES_PASSWORD=superset
-POSTGRES_PORT=5432
-POSTGRES_USER=superset
+
+# database configurations (do not modify)
+DATABASE_DB=superset
+DATABASE_HOST=db
+DATABASE_PASSWORD=superset
+DATABASE_USER=superset
+
+# database engine specific environment variables
+# change the below if you prefers another database engine
+DATABASE_PORT=5432
+DATABASE_DIALECT=postgresql
+POSTGRES_DB=${DATABASE_DB}
```

Comment:
Why the extra level of indirection here? Are these vars referenced anywhere else? I see that you also updated `docker/pythonpath_dev/superset_config.py` to reference the `DATABASE_XXX` vars. I think we should just go for `DATABASE_XXX` all the way. 😄

Reply:
To my understanding, the PostgreSQL image uses the `POSTGRES_XXX` variables to initialize the database. The `DATABASE_XXX` variables are used by SQLAlchemy (Flask in Python) to communicate with the database. This abstraction is to avoid the confusion having to define the variables multiple time when switching the database engine.

Reply:
Ok, that makes sense